twine/commands/upload.py

```diff
@@ -48,6 +50,40 @@ def skip_upload(response, skip_existing, package):
             (response.status_code == 403 and msg_403 in response.text)))
 
 
+def check_status_code(response, verbose):
+    """
+    Additional safety net to catch response code 410 in case the
```

Comment:
This also doesn't follow PEP257

Reply:
Most of the rest of the code follows PEP 257 iirc (or at least I was updating it along the way to ensure it did). I suspect we've had enough regressions that it'll be painful to add that in now.